lib/deaf_grandma.rb

```diff
@@ -19,10 +20,19 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+     
+     if input != input.upcase
+          "SPEAK UP SONNY!"
+    elsif input == "BYE"
+             @bye_counter += 1
+                if @bye_counter == 3
+                     "SEE YOU LATER SONNY!"
+                exit
+                end
+    else
+        "NOT SINCE 1964!"
```

Comment:
Consistent indentation makes it easier to review and for the next developer to work on.

https://github.com/bbatsov/ruby-style-guide#source-code-layout